app/views/topical_events/show.html.erb

```diff
@@ -59,4 +65,33 @@
       <% end %>
     <% end %>
   </div>
+
+  <div class="govuk-grid-column-two-thirds">
+    <% if @topical_event.organisations %>
+          <div class="organisations">
+            <%= render "govuk_publishing_components/components/heading", {
+              text: t("topical_events.organisations"),
+              font_size: "l",
+              margin_bottom: 4,
+              padding: true,
+              border_top: 2,
+            } %>
+
+            <div class="govuk-grid-row">
+              <% @topical_event.organisations.each do |organisation| %>
+                <div class="govuk-grid-column-one-quarter govuk-!-padding-bottom-7">
```

Comment:
The layout seems to go a bit wobbly when there are more than 4 images (e.g. for [this topical event](http://collections.dev.gov.uk/government/topical-events/russian-invasion-of-ukraine-uk-government-response)).

![Screenshot of the "Who's involved" section showing five organisation logos.  The first four are on a single line. The fifth is on the following line but aligned to the right.](https://user-images.githubusercontent.com/6329861/174582212-6594eebf-214b-4df1-ae18-3c0e5d41cac1.png)